example/models_directory/configuration_interaction.json

```diff
@@ -0,0 +1,28 @@
+{
+    "tier1": "physics-based",
+    "tier2": {
+        "name": "quantum-mechanical",
+        "slug": "quantummechanical"
+    },
+    "tier3": {
+        "name": "ab initio",
+        "slug": "abinitio"
```

Comment:
"abin" for brevity?

Reply:
https://github.com/Exabyte-io/esse/pull/139/files#diff-e9574187048c782764567d74ba0fdcdeR8